client/components/App.js

```diff
@@ -11,7 +12,13 @@ const App = props => {
     $(document).foundation()
   }, [])
 
-  return <div className="callout primary">Replace this div with your Router.</div>
+  return <div className="callout primary">
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" component={PetTypesPage} />
```

Comment:
Let's include `exact` on this `Route` and we'll need a `<Route>` accounting for `/pet-types`